autocleaner.py

```diff
@@ -0,0 +1,295 @@
+#!/bin/python3
+import os
+import datetime
+import subprocess
+import re
+import argparse
+import logging
+import sys
+import traceback
+import shutil
+import time
+from multiprocessing import Process
+
+import psutil
+import ruamel.yaml
+import requests
+from systemd.journal import JournalHandler
+
+from tasks.constants import JOBS_DIR, UUID_RE
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+consoleHandler = logging.StreamHandler()
+consoleHandler.setLevel(logging.INFO)
+logger.addHandler(consoleHandler)
+logger.addHandler(JournalHandler())
+logger.setLevel(logging.DEBUG)
+
+PRCI_CONFIG = '/root/.config/freeipa-pr-ci/config.yml'
+PRCI_DEF_DIR = 'ipatests/prci_definitions'
```

Comment:
shouldn't be these constants defined into tasks.constants? or another constants.py file? I guess it will make sense if they are used in other places as well.